plugins/driplink/plugin.js

```diff
@@ -91,6 +111,11 @@
 					command: 'unlink',
 					toolbar: 'links,20'
 				} );
+	      editor.ui.addButton( 'DripButton', {
```

Comment:
Not sure we need to namespace here with `Drip`, since the others are simply called "Link" and "Unlink". Let's call it `ButtonLink` instead, the a command of `buttonlink`. Also, looks like `toolbar` is `help` here while the others are `links,...`. Any particular reason for that?